example/src/views/CheckboxView.vue

```diff
@@ -2,9 +2,7 @@
 import { type CheckboxProps, RuiCheckbox } from '@rotki/ui-library-compat';
 import { ref } from 'vue';
 
-type CheckboxData = CheckboxProps & {
-  value?: boolean;
-};
+type CheckboxData = CheckboxProps;
```

Comment:
since we're not adding any extra data, maybe remove and use `CheckboxProps` directly